ex-1.sc

```diff
@@ -0,0 +1,20 @@
+def examWords(args: List[String]): Unit = {
```

Comment:
The solution is for sure clean and straightforward in term statements, but the approach is imperative and computation and presentation are mixed.
There is one other factor that are the performance.
Did you think about the time complexity? How much do you think it is?
How many time the code iterate on the list?